### ISSUE TITLE
Cannot read property 'TIMESTAMPS' of undefined

### ISSUE THREAD

**Issue by meshekhar** [firebase/emberfire#391]
I am getting "Cannot read property 'TIMESTAMPS' of undefined" when try to create a timestamp using **firebase.database.ServerValue.TIMESTAMPS**.

Version:
Ember          : 2.5.1
Ember Data : 2.6.1
Firebase      : 3.0.5
EmberFire   : 2.0.0

```
      const fb = this.get('firebaseApp');
      fb.auth().createUserWithEmailAndPassword(email, password)
        .then((userResponse) => {
          const user = this.store.createRecord('user', {
            uid: userResponse.uid,
            username: username,
            createdAt: fb.database.ServerValue.TIMESTAMPS
          });
          return user.save();
        }).catch(function(error) {
          // Handle Errors here.
          var errorCode = error.code;
          var errorMessage = error.message;
          console.log("Error: ",error.code, " | ", error.message);
        });
```
Error:  undefined  |  Cannot read property 'TIMESTAMPS' of undefined

**Comment by tstirrat** [comment]
Hey, its singular `firebase.database.ServerValue.TIMESTAMP`

Also, you cannot use the `firebaseApp` service, use import instead:

```js
import firebase from 'firebase';

firebase.database.ServerValue.TIMESTAMP;
```

See: https://firebase.google.com/docs/database/web/offline-capabilities#server-timestamps

The `firebaseApp` service is the actual app (e.g. `firebase.app()`, not the global `firebase` var.)

**Comment by meshekhar** [comment]
Sorry for the silly mistake.

I implemented your suggestion but still can't get the timestamp to populate the db.

Below is my user signup controller:

````
import Ember from 'ember';
import firebase from 'firebase';

export default Ember.Controller.extend({

  firebaseApp: Ember.inject.service(),

  actions: {
    signUp(provider, email, password, username){
      const fb = this.get('firebaseApp');
      fb.auth().createUserWithEmailAndPassword(email, password)
        .then((userResponse) => {
          const user = this.store.createRecord('user', {
            uid: userResponse.uid,
            createdAt: firebase.database.ServerValue.TIMESTAMP,
            username: username
          });
          return user.save();
        }).catch(function(error) {
          // Handle Errors here.
          //var errorCode = error.code;
          //var errorMessage = error.message;
          console.log("Error: ",error.code, " | ", error.message);
        });
  },
}
});
````

User model:
````
import DS from 'ember-data';
export default DS.Model.extend({
  uid: DS.attr('string'),
  createdAt: DS.attr('number'),
  username: DS.attr('string')
});
````

In firebase database I get the following. createdAt is missing.
uid: "a4hoG4mW6gfLccyc2732k7C9N6F2"
username: "qwewfsafsad"

Note: If I use ``` new Date().getTime(), ``` instead of firebase timestamp it works fine.

**Comment by jamesdaniels** [comment]
Addressed in #448
